tutorials/euclid_access/2_Euclid_intro_MER_catalog.md

```diff
@@ -92,14 +92,22 @@ for col in columns:
     print(f'{f"{col.name}":30s}  {col.unit}  {col.description}')
 ```
 
+## Note that the mer catalog contains 476 columns, below are a few highlights:
```

Comment:
You have seemed to branched off of your other branch rather than from `caltech-ipac/main` :)